src/frontend/komponenter/Fagsak/journalposter/JournalpostListe.tsx

```diff
@@ -253,7 +286,20 @@ const JournalpostListe: React.FC<IProps> = ({ bruker }) => {
                                         size="small"
                                         title={journalpost.avsenderMottaker?.navn}
                                     >
-                                        {journalpost.avsenderMottaker?.navn}
+                                        {journalpost.utsendingsinfo &&
+                                        toggles[ToggleNavn.journalpostUtsendingsinfo] ? (
+                                            <Link
```

Comment:
Dette er ikke en reell lenke, så `<Button variant="tertiary">` blir riktigere

Reply:
Vet, men brukte lenke for at det skulle matche "lenken" som åpner PdfVisningModal. Blir ikke spesielt pent med den ene endret til Button, synes jeg:
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/47184872/55fb318f-b56f-46c7-a5ae-2d262518f1ae)

Reply:
Ble forsåvidt litt penere med button size 'xsmall', så jeg endret til det 👍 
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/47184872/cf2801c1-7674-44fe-ba91-877752d699fa)